test/factories/surveys.rb

```diff
@@ -2,18 +2,17 @@
 #
 # Table name: surveys
 #
-#  id          :integer          not null, primary key
-#  name        :string
-#  published   :boolean
-#  user_id     :integer
-#  created_at  :datetime         not null
-#  updated_at  :datetime         not null
-#  uuid        :string           not null
-#  description :text
```

Comment:
Where did description go?

Reply:
Something odd is happening when I run `rails db:migrate` it keeps changing this annotated part each time, even though I am not changing those models.

Reply:
For future reference this is a known issue; https://github.com/ctran/annotate_models/issues/541